knowledge/studios/bbc_studios/qna.yaml

```diff
@@ -0,0 +1,24 @@
+created_by: 
```

Comment:
Please add your GitHub username here, then we can run the linter.

Reply:
Added github username